src/main/java/com/capgemini/web/authentication/AuthenticationHelper.java

```diff
@@ -30,4 +30,13 @@ public static String getCurrentUsername(){
         User currentUser = (User) auth.getPrincipal();
         return currentUser.getUsername();
     }
+
+    public static boolean isLoggedIn(){
+        Authentication auth = SecurityContextHolder.getContext().getAuthentication();
+        if (auth.getPrincipal() == "anonymousUser"){
```

Comment:
1. Strings vergelijken met .equals() graag, dit is geen garantie

2. Niet zo belangrijk, maar je zou ook het hele if else statement weg kunnen halen en het volgende doen:
return auth.getPrincipal().equals("anonymousUser");